crates/http-server/src/cli.rs

```diff
@@ -9,12 +9,12 @@ pub struct Cli {
     #[arg(long, default_value = "127.0.0.1")]
     pub host: IpAddr,
     /// The port to listen on
-    #[arg(long, default_value = "7878")]
+    #[arg(long, default_value = "3000")]
```

Comment:
Can we keep this as `7878`? Which is the one used in [The Rust Programming Language Book](https://doc.rust-lang.org/book/ch20-01-single-threaded.html#listening-to-the-tcp-connection)?

> Using TcpListener, we can listen for TCP connections at the address 127.0.0.1:7878. In the address, the section before the colon is an IP address representing your computer (this is the same on every computer and doesn’t represent the authors’ computer specifically), and 7878 is the port. We’ve chosen this port for two reasons: HTTP isn’t normally accepted on this port so our server is unlikely to conflict with any other web server you might have running on your machine, and 7878 is rust typed on a telephone.